src/clr-addons/notification/notification.service.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ClrNotificationService {
+  elements = [];
+
```

Comment:
we also need an addElements here, otherwise any component needs to know the elements of every other component